shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/AboutConfScreen.kt

```diff
@@ -362,17 +351,7 @@ private fun AboutConferenceFooter(
         ) {
             Text(
                 buildAnnotatedString {
-                    append("You can find more information about the conference on the official website:")
-                },
-                style = MaterialTheme.typography.body2.copy(color = grey50),
-                modifier = Modifier
-                    .clickable {
-                        uriHandler.openUri("https://kotlinconf.com")
-                    }
-            )
-
-            Text(
-                buildAnnotatedString {
+                    append("You can find more information about the conference on the official website:\n")
                     withStyle(SpanStyle(textDecoration = TextDecoration.Underline)) {
                         append("kotlinconf.com")
```

Comment:
Don't you want to extract it to the strings resources?

Reply:
yep, fixed